<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QtPastePicClass</class>
 <widget class="QMainWindow" name="QtPastePicClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1155</width>
    <height>921</height>
   </rect>
  </property>
  <property name="mouseTracking">
   <bool>true</bool>
  </property>
  <property name="tabletTracking">
   <bool>true</bool>
  </property>
  <property name="focusPolicy">
   <enum>Qt::WheelFocus</enum>
  </property>
  <property name="acceptDrops">
   <bool>true</bool>
  </property>
  <property name="windowTitle">
   <string>QtPastePic</string>
  </property>
  <property name="autoFillBackground">
   <bool>true</bool>
  </property>
  <property name="toolButtonStyle">
   <enum>Qt::ToolButtonFollowStyle</enum>
  </property>
  <property name="documentMode">
   <bool>true</bool>
  </property>
  <property name="dockNestingEnabled">
   <bool>true</bool>
  </property>
  <property name="unifiedTitleAndToolBarOnMac">
   <bool>true</bool>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QScrollArea" name="source_pic_show">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>50</y>
      <width>250</width>
      <height>188</height>
     </rect>
    </property>
    <property name="widgetResizable">
     <bool>false</bool>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <widget class="QWidget" name="scrollAreaWidgetContents">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>39</y>
       <width>229</width>
       <height>109</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QScrollArea" name="target_pic_show">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>300</y>
      <width>250</width>
      <height>188</height>
     </rect>
    </property>
    <property name="widgetResizable">
     <bool>false</bool>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <widget class="QWidget" name="scrollAreaWidgetContents_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>49</y>
       <width>229</width>
       <height>89</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="source_pic_lable">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>20</y>
      <width>101</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Times New Roman</family>
      <pointsize>14</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Source Pic</string>
    </property>
    <property name="buddy">
     <cstring>source_pic_load_button</cstring>
    </property>
   </widget>
   <widget class="QLabel" name="target_pic_lable">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>270</y>
      <width>111</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Times New Roman</family>
      <pointsize>14</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Target Pic</string>
    </property>
    <property name="buddy">
     <cstring>target_pic_load_button</cstring>
    </property>
   </widget>
   <widget class="QPushButton" name="source_pic_load_button">
    <property name="geometry">
     <rect>
      <x>730</x>
      <y>20</y>
      <width>121</width>
      <height>23</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Times New Roman</family>
      <pointsize>10</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>Source Pic Load </string>
    </property>
   </widget>
   <widget class="QPushButton" name="target_pic_load_button">
    <property name="geometry">
     <rect>
      <x>730</x>
      <y>270</y>
      <width>121</width>
      <height>23</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Times New Roman</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Target Pic Load </string>
    </property>
   </widget>
   <widget class="QPushButton" name="LittleButton">
    <property name="geometry">
     <rect>
      <x>881</x>
      <y>570</y>
      <width>250</width>
      <height>27</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Times New Roman</family>
      <pointsize>12</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>Zoom Out</string>
    </property>
   </widget>
   <widget class="QPushButton" name="BigButton">
    <property name="geometry">
     <rect>
      <x>881</x>
      <y>535</y>
      <width>250</width>
      <height>27</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Times New Roman</family>
      <pointsize>12</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>Zoom In</string>
    </property>
   </widget>
   <widget class="QPushButton" name="UpButton">
    <property name="geometry">
     <rect>
      <x>881</x>
      <y>605</y>
      <width>250</width>
      <height>27</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Times New Roman</family>
      <pointsize>12</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>Up</string>
    </property>
   </widget>
   <widget class="QPushButton" name="DownButton">
    <property name="geometry">
     <rect>
      <x>881</x>
      <y>640</y>
      <width>250</width>
      <height>27</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Times New Roman</family>
      <pointsize>12</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>Down</string>
    </property>
   </widget>
   <widget class="QPushButton" name="RightButton">
    <property name="geometry">
     <rect>
      <x>881</x>
      <y>710</y>
      <width>250</width>
      <height>27</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Times New Roman</family>
      <pointsize>12</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>Right</string>
    </property>
   </widget>
   <widget class="QPushButton" name="LiftButton">
    <property name="geometry">
     <rect>
      <x>881</x>
      <y>675</y>
      <width>250</width>
      <height>27</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Times New Roman</family>
      <pointsize>12</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>Left</string>
    </property>
   </widget>
   <widget class="QPushButton" name="source_mask_load_button">
    <property name="geometry">
     <rect>
      <x>1010</x>
      <y>20</y>
      <width>121</width>
      <height>23</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Times New Roman</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Segment</string>
    </property>
   </widget>
   <widget class="QScrollArea" name="source_mask_show">
    <property name="geometry">
     <rect>
      <x>880</x>
      <y>50</y>
      <width>250</width>
      <height>188</height>
     </rect>
    </property>
    <property name="widgetResizable">
     <bool>false</bool>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <widget class="QWidget" name="scrollAreaWidgetContents_3">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>39</y>
       <width>229</width>
       <height>109</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="target_mask_load_button">
    <property name="geometry">
     <rect>
      <x>1010</x>
      <y>270</y>
      <width>121</width>
      <height>23</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Times New Roman</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Segment</string>
    </property>
   </widget>
   <widget class="QLabel" name="source_pic_lable_2">
    <property name="geometry">
     <rect>
      <x>880</x>
      <y>20</y>
      <width>131</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Times New Roman</family>
      <pointsize>14</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Source Mask</string>
    </property>
    <property name="buddy">
     <cstring>source_pic_load_button</cstring>
    </property>
   </widget>
   <widget class="QScrollArea" name="target_mask_show">
    <property name="geometry">
     <rect>
      <x>880</x>
      <y>300</y>
      <width>250</width>
      <height>188</height>
     </rect>
    </property>
    <property name="widgetResizable">
     <bool>false</bool>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <widget class="QWidget" name="scrollAreaWidgetContents_4">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>49</y>
       <width>229</width>
       <height>89</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="target_mask_lable">
    <property name="geometry">
     <rect>
      <x>880</x>
      <y>270</y>
      <width>131</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Times New Roman</family>
      <pointsize>14</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Target Mask</string>
    </property>
    <property name="buddy">
     <cstring>target_pic_load_button</cstring>
    </property>
   </widget>
   <widget class="QLabel" name="target_depth_lable">
    <property name="geometry">
     <rect>
      <x>599</x>
      <y>522</y>
      <width>121</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Times New Roman</family>
      <pointsize>14</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Target Depth</string>
    </property>
    <property name="buddy">
     <cstring>target_pic_load_button</cstring>
    </property>
   </widget>
   <widget class="QPushButton" name="target_depth_load_button">
    <property name="geometry">
     <rect>
      <x>730</x>
      <y>522</y>
      <width>121</width>
      <height>23</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Times New Roman</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Depth Estimation</string>
    </property>
   </widget>
   <widget class="QScrollArea" name="target_depth_show">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>552</y>
      <width>250</width>
      <height>188</height>
     </rect>
    </property>
    <property name="widgetResizable">
     <bool>false</bool>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <widget class="QWidget" name="scrollAreaWidgetContents_5">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>49</y>
       <width>229</width>
       <height>89</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QScrollArea" name="output_show">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>470</y>
      <width>550</width>
      <height>380</height>
     </rect>
    </property>
    <property name="lineWidth">
     <number>0</number>
    </property>
    <property name="widgetResizable">
     <bool>false</bool>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <widget class="QWidget" name="scrollAreaWidgetContents_6">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>548</width>
       <height>378</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="output">
    <property name="geometry">
     <rect>
      <x>880</x>
      <y>780</y>
      <width>250</width>
      <height>50</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Times New Roman</family>
      <pointsize>16</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Make Pic</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>440</y>
      <width>91</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Times New Roman</family>
      <pointsize>14</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Output</string>
    </property>
   </widget>
   <widget class="QPushButton" name="mask">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>780</y>
      <width>250</width>
      <height>50</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Times New Roman</family>
      <pointsize>16</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Make Mask</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>91</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Times New Roman</family>
      <pointsize>14</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Painter</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1155</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="QtPastePic.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>LittleButton</sender>
   <signal>clicked()</signal>
   <receiver>QtPastePicClass</receiver>
   <slot>on_little_clicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>1010</x>
     <y>133</y>
    </hint>
    <hint type="destinationlabel">
     <x>464</x>
     <y>505</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>UpButton</sender>
   <signal>clicked()</signal>
   <receiver>QtPastePicClass</receiver>
   <slot>on_up_clicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>1010</x>
     <y>163</y>
    </hint>
    <hint type="destinationlabel">
     <x>517</x>
     <y>569</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>DownButton</sender>
   <signal>clicked()</signal>
   <receiver>QtPastePicClass</receiver>
   <slot>on_down_clicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>1010</x>
     <y>193</y>
    </hint>
    <hint type="destinationlabel">
     <x>490</x>
     <y>534</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>LiftButton</sender>
   <signal>clicked()</signal>
   <receiver>QtPastePicClass</receiver>
   <slot>on_left_clicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>1010</x>
     <y>223</y>
    </hint>
    <hint type="destinationlabel">
     <x>476</x>
     <y>586</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>RightButton</sender>
   <signal>clicked()</signal>
   <receiver>QtPastePicClass</receiver>
   <slot>on_right_clicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>1010</x>
     <y>253</y>
    </hint>
    <hint type="destinationlabel">
     <x>460</x>
     <y>616</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>BigButton</sender>
   <signal>clicked()</signal>
   <receiver>QtPastePicClass</receiver>
   <slot>on_big_clicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>1010</x>
     <y>103</y>
    </hint>
    <hint type="destinationlabel">
     <x>462</x>
     <y>454</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>on_source_pic_load_button_clicked()</slot>
  <slot>on_target_pic_load_button_clicked()</slot>
  <slot>on_up_clicked()</slot>
  <slot>on_big_clicked()</slot>
  <slot>on_little_clicked()</slot>
  <slot>on_down_clicked()</slot>
  <slot>on_left_clicked()</slot>
  <slot>on_right_clicked()</slot>
 </slots>
</ui>
